tests/scripts/set-release-ver.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
```

Comment:
Have you tried running this script on your mac? It doesn't seem to work for me, the files are not updated in place by `sed`.

Reply:
I was wondering when/if this would come up. Mac's sed version is POSIX/BSD sed, not gnu-sed, and the -i option is different in gnu.

This may need the `-i ''` option as noted here: https://www.baeldung.com/linux/gnu-bsd-stream-editor#2-the--i-option

Reply:
I tried it but the main problem currently seems to be that sed refuses top operate on non-regular filel like directory  `deploy/examples/cosi` which are expanded by `deploy/examples/*`.

I will figure it uot and update.

Reply:
FWIW: works just fine for me on Linux where I developed it initially. will take care of MacOS next.

Reply:
MacOS works again now.

Reply:
@BlaineEXE  wrote:
> I was wondering when/if this would come up. Mac's sed version is POSIX/BSD sed, not gnu-sed, and the -i option is different in gnu.
> 
> This may need the `-i ''` option as noted here: https://www.baeldung.com/linux/gnu-bsd-stream-editor#2-the--i-option

after struggling with a correct invocation  for a while, I ended up uysing the `build/sed-in-place` script.